### PR TITLE
HelpScreen Component

### PR DESCRIPTION
**Pull request by kxngnathan**
This Screen Component allows a user request for help as well as report issues encountered on the application

- Created SupportScreen and added Buttons 
- Created a SupportNavigator
- Created  ReporNavigator(Report issues)  TopTab component containing pages POS and Other Issues screens 
- Created Other Issues screen component (Not complete)
- Added Report Navigator to SupportNavigator
- Added SupportNavigator to Home Drawer component

- git checkout nathan
- click drawer icon 
- navigate to help center

Screenshot_20200226-123309_Expo
Screenshot_20200226-123519_Expo
Screenshot_20200226-123524_Expo

### CODE REVIEW

File: src/screens/Support/OtherIssues.tsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    </Block>
+  )
+
+
+}
+
+}
+
+
+
```
Comment (dbugshe2) [inline src/screens/Support/OtherIssues.tsx]: lol, you forgot to delete these

File: src/screens/Support/SupportScreen.tsx [hunk 2]
```diff
@@ -1,12 +1,217 @@
...
+                  flexDirection: 'row',
+                  flexWrap: 'wrap',
+                  // alignItems: '',
+                  paddingTop: 30
+              }}
+              >
+
+              <Button
+                  white
+                  //   flex={0.4}
```
Comment (dbugshe2) [inline src/screens/Support/SupportScreen.tsx]: we are advised to delete commented code

**Review by dbugshe2 — CHANGES_REQUESTED** [review]

File: src/screens/Support/OtherIssues.tsx (hunk 1 above)
Comment (kxngnathan) [reply src/screens/Support/OtherIssues.tsx]: lol error

Comment (Charpell) [reply src/screens/Support/OtherIssues.tsx]: @kxngnathan
Have uou made the changes

Comment (kxngnathan) [reply src/screens/Support/OtherIssues.tsx]: comments removed thank you @dbugshe2

File: src/screens/Support/SupportScreen.tsx (hunk 2 above)
Comment (kxngnathan) [reply src/screens/Support/SupportScreen.tsx]: Comments removed @dbugshe2

**Pull request merged**